vwifi.c

```diff
@@ -57,6 +57,9 @@ struct owl_ndev_priv_context {
     struct wireless_dev wdev;
     struct net_device *ndev;
     struct net_device_stats stats;
+    bool is_assoc;             /* whether the station is associated with an AP*/
+    unsigned long conn_time;   /* last connection time to a AP (in jiffies) */
+    unsigned long active_time; /* last tx/rx time (in jiffies) */
```

Comment:
Above three lines is layer 2 information. Not relative to the data path. Could we move wifi management information to `owl_context`?

Reply:
OK. Sure.